RELEASING.md

```diff
@@ -45,18 +46,20 @@ release is announced.
 
 ### Promoting a release candidate to a final release
 1. Add a new `<option>` entry to the `<span class="dropdown">` in `_layouts/docwithnav.html` file. This step should NOT be performed until testing of the release is complete.
-2. Modify the redirect in `/index.html` to point to your new release.. 
-3. Move the section for the release in `_data/versions.yaml` to the top of the file so that it will be the 'Latest Release'.
-4. Run `make add_redirects_for_latest VERSION=vX.Y` to update the redirects.
-2. Test the changes locally then open a pull request, make sure it passes CI and get it reviewed.
-3. Create a git tag and docker images by running `make -C calico_node release`. Follow the instructions to push the images but DO NOT PUSH THE TAG YET.
-4. Now merge the PR - this will cause the live docs site to be updated (after a few minutes).
-5. Finally, push the tag.
+1. Modify the redirect in `/index.html` to point to your new release.. 
+1. Move the section for the release in `_data/versions.yaml` to the top of the file so that it will be the 'Latest Release'.
+1. Run `make add_redirects_for_latest VERSION=vX.Y` to update the redirects.
+1. Test the changes locally then open a pull request, make sure it passes CI and get it reviewed.
+1. Create a git tag and docker images by running `make -C calico_node release`. Follow the instructions to push the images but DO NOT PUSH THE TAG YET.
```

Comment:
This is where the release tar is created, right? Can you mention it by saying:

Create a git tag, docker images, and release tar by running...

Right?